.github/workflows/ci.yml

```diff
@@ -173,7 +174,11 @@ jobs:
             echo helm package for $directory
             name=$(cat $directory/Chart.yaml | grep ^name: | cut -d: -f2 | cut -c 2- | tr -d '"')
             chart_version=$(cat $directory/Chart.yaml | grep ^version: | cut -d: -f2 | cut -c 2- | tr -d '"')
-            release_version=$chart_version-${{ env.VERSION }}
+            if [[ "${{ env.VERSION }}" == *develop* || "${{ env.VERSION }}" == *master* || "${{ env.VERSION }}" != [0-9]* ]]; then
+              release_version=$chart_version-${{ env.VERSION }}:${{ env.COMMIT_HASH }}
+            else
+              release_version=$chart_version-${{ env.VERSION }}
+            fi
```

Comment:
Why do we need `|| "${{ env.VERSION }}" != [0-9]*`?

Reply:
You can use `${{ github.event.release.prerelease }} ` instead

Reply:
1) And are you sure we can use : as a separator here? Maybe simple dash - is better, isn't it?
2) Could you use explicitly 'master' and 'develop'? No need to pass 'master2', '2master', `my-best-master`.